h/api/views.py

```diff
@@ -228,7 +228,10 @@ def delete(annotation, request):
     # process the delete event. For example, the streamer needs to know the
     # target URLs of the deleted annotation in order to know which clients to
     # forward the delete event to.
-    _publish_annotation_event(request, annotation, 'delete')
+    _publish_annotation_event(
+        request,
+        AnnotationJSONPresenter(request, annotation).asdict(),
```

Comment:
`annotation` is sometimes a Pg annotation not an Es one now, and Pg annotations aren't JSON serializable, so present the annotation and call the presenter's `asdict()`, this works with both Pg and Es annotations.